playground/backend/internal/preparers/preparer_test.go

```diff
@@ -0,0 +1,182 @@
+	pyGraphFile     = "graphFile.py"
+)
+
+func TestMain(m *testing.M) {
+	err := setupPreparedFiles()
+	if err != nil {
+		logger.Fatal(err)
+	}
+	defer teardown()
+	m.Run()
+}
+
+// setupPreparedFiles creates 2 go programs:
+// correctFile - program without errors
+// incorrectFile - program with errors
+func setupPreparedFiles() error {
```

Comment:
It seems that this method creates more than 2 go programs

Reply:
Changed description